.github/pr_labeler.yml

```diff
@@ -0,0 +1,22 @@
+"C:CLI":
```

Comment:
Lets also create a label for the x/evm and x/marketplace module

ps: you also need to create the label on the repo to make it works but I can do for you if you dont have the permission